tests/integration/offer/test_applicator.py

```diff
@@ -73,6 +73,21 @@ def test_get_site_offers(self):
         self.assertEqual(len(site_offers), 1)
         self.assertEqual(site_offers[0].name, "globaloffer")
 
+    def test_apply_offer_with_multibuy_benefit_and_count_condition(self):
```

Comment:
I this test should go in `tests/integration/offer/test_multibuy_benefit.py` - where we have tests for other types of conditions.

Reply:
Hi, sure. I will move it.